core/templates/domain/editor/undo_redo/ChangeObjectFactory.ts

```diff
@@ -26,69 +26,80 @@ import cloneDeep from 'lodash/cloneDeep';
 import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable } from '@angular/core';
 
+import { IMisconceptionBackendDict } from
+  'domain/skill/MisconceptionObjectFactory';
+
+interface IBackendChangeObject {
```

Comment:
Have you verified that all the new structures editors work as well?

Reply:
I don't think there should be problems on the behaviour with just adding the types, as the code is transpiled to js anyway.